pages/index.html

```diff
@@ -0,0 +1,66 @@
+---
+layout: homepage
+title: Homepage
+---
+
+<article>
+  <h2 class="first-grid-item">
+    CBA is an extension for chrome browser automation
+  </h2>
```

Comment:
You can move this into homepage.ejs and use `page.title`, as the value.

Reply:
If we change `page.title`'s value it will change the Homepage button title as well on navigation menu. Do you want to change it to "CBA is an extension for chrome browser automation"?

Reply:
You can introduce: a new front-matter property ex.: `menu`. so `title` will work as a fallback for menu items that doesn't have that prop.

Reply:
THis way we will be consistent with Druapal, as if you notice in Drupal version, the title of the page is `CBA is an extension for chrome browser automation ` instead of `homepage`.

Reply:
> 
> 
> You can move this into homepage.ejs and use `page.title`, as the value.

Ok, but did you mean outside of `article` wrapper? If so there would be no need for `grid-column` property for first-child anymore, because it is no child of `article` wrapper anymore.

Reply:
From [MDN ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) : 

> `Each <article> should be identified, typically by including a heading (<h1>-<h6> element) as a child of the <article> element`

So title should be located inside of `<article>`, so you might want to move `<article>` as well, that way you also will be consistent with the `default.ejs`(I guess).